### DISCUSSION TITLE
Install using conda : alm work ,but anphon don't work

### DISCUSSION THREAD

**Discussion by cancer521** [ttadano/alamode#146] [Q&A]
[thank you all]：
ALAMODE version: 1.4.2
anphon phband.in > GeTe_phband.log
[DESKTOP-IA8RQKB:21039] *** Process received signal ***
[DESKTOP-IA8RQKB:21039] Signal: Segmentation fault (11)
[DESKTOP-IA8RQKB:21039] Signal code: Address not mapped (1)
[DESKTOP-IA8RQKB:21039] Failing at address: 0x440000e8
[DESKTOP-IA8RQKB:21039] [ 0] /lib/x86_64-linux-gnu/libc.so.6(+0x42520)[0x151acb22e520]
[DESKTOP-IA8RQKB:21039] [ 1] /home/caner/anaconda3/envs/alamode/lib/libmpi.so.40(PMPI_Comm_rank+0x37)[0x151acb65ead7]
[DESKTOP-IA8RQKB:21039] [ 2] anphon(_ZN7PHON_NS5MyMPIC1EPNS_4PHONEi+0x10f)[0x55aef0b9b11f]
[DESKTOP-IA8RQKB:21039] [ 3] anphon(_ZN7PHON_NS4PHONC1EiPPci+0x76)[0x55aef0bb5df6]
[DESKTOP-IA8RQKB:21039] [ 4] anphon(main+0x41)[0x55aef0b14a01]
[DESKTOP-IA8RQKB:21039] [ 5] /lib/x86_64-linux-gnu/libc.so.6(+0x29d90)[0x151acb215d90]
[DESKTOP-IA8RQKB:21039] [ 6] /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0x80)[0x151acb215e40]
[DESKTOP-IA8RQKB:21039] [ 7] anphon(+0x2ccf5)[0x55aef0b14cf5]
[DESKTOP-IA8RQKB:21039] *** End of error message ***
Segmentation fault

**Comment by ALukmantoro16** [comment]
Hii
Have you solved this problem? I also get same problem lik this in my computer
thanks

**Comment by 16-vikrant** [comment]
Follow this for the solution. 


https://github.com/ttadano/alamode/discussions/121
